gradle/libs.versions.toml

```
@@ -37,7 +37,7 @@ gmsPlugin = "4.4.0"
 googleOss = "17.0.1"
 googleOssPlugin = "0.10.6"
 hilt = "2.48.1"
-hiltExt = "1.0.0"
+hiltExt = "1.1.0-alpha01"
```

Stable version is ready live: https://developer.android.com/jetpack/androidx/releases/hilt#1.1.0
```suggestion
hiltExt = "1.1.0"
```

---

Done